tools/cli/src/webpack/index.ts

```diff
@@ -343,7 +343,7 @@ export function createWebpackConfig(
     devServer: {
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +291]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->remove it<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/9286?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->